nodes-lib/src/config/chain.ts

```diff
@@ -5,16 +5,19 @@ import { type NodesEnv } from "./index.js";
 export type NodesContract =
   | tc.ResearchObject
   | tc.ResearchObjectV2
-  | tc.DpidRegistry;
+  | tc.DpidRegistry
+  | tc.DpidAliasRegistry;
 
 export type ContractConnector<T extends NodesContract> =
   (signerOrProvider: Signer | providers.Provider) => T;
 
 export type ChainID =
-  | "1337"
-  | "11155111";
+  | "1337" // local
```

Comment:
consider making these named types / enums if possible for future clarity / reuse